src/storage/mvcc/mod.rs

```diff
@@ -158,6 +158,12 @@ pub enum ErrorInner {
         existing_commit_ts: TimeStamp,
     },
 
+    #[error(
+        "Lock_only_if_exists of a pessimistic lock request is set to true, but return_value is not, key:{}",
+        log_wrappers::Value::key(.key)
+    )]
+    LockIfExistsFailed { key: Vec<u8> },
```

Comment:
The `start_ts` needs to be recorded to identify the corresponding transaction and logs.